doltpy/cli/dolt.py

```diff
@@ -444,92 +429,6 @@ def _parse_tabluar_output_to_dict(self, args: List[str]):
-
-            proc = Popen(args=['dolt'] + server_args,
-                         cwd=self.repo_dir(),
-                         stdout=open(log_file, 'w'),
-                         stderr=STDOUT)
-
-            self.server = proc
-
-        args = ['sql-server']
-
-        if self.server_config.config:
-            args.extend(['--config', self.server_config.config])
-        else:
-            if self.server_config.host:
-                args.extend(['--host', self.server_config.host])
-            if self.server_config.port:
-                args.extend(['--port', str(self.server_config.port)])
-            if self.server_config.user:
-                args.extend(['--user', self.server_config.user])
-            if self.server_config.password:
-                args.extend(['--password', self.server_config.password])
-            if self.server_config.timeout:
-                args.extend(['--timeout', str(self.server_config.timeout)])
-            if self.server_config.readonly:
-                args.extend(['--readonly'])
-            if self.server_config.loglevel:
-                args.extend(['--loglevel', self.server_config.loglevel])
-            if self.server_config.multi_db_dir:
-                args.extend(['--multi-db-dir', self.server_config.multi_db_dir])
-            if self.server_config.no_auto_commit:
-                args.extend(['--no-auto-commit'])
-
-        start_server(args)
-
-    @property
-    def repo_name(self):
-        return str(self.repo_dir()).split('/')[-1].replace('-', '_')
-
-    def _get_engine(self) -> Engine:
-        """
-        Get a connection to ths server process that this repo is running, raise an exception if it is not running.
-        :param echo:
-        :return:
-        """
-        database = self.repo_name
-        host = self.server_config.host
-        port = self.server_config.port
-
-        logger.info('Creating engine for Dolt SQL Server instance running on {}:{}'.format(host, port))
-
-        def inner():
-            return create_engine('mysql+mysqlconnector://{user}@{host}:{port}/{database}'.format(user='root',
-                                                                                                 host=host,
-                                                                                                 port=port,
-                                                                                                 database=database),
-                                 echo=self.server_config.echo)
-
-        return inner()
-
-    def get_engine(self):
-        return self.engine
-
-    def sql_server_stop(self):
-        """
-        Stop the MySQL Server process this repo is running.
-        :return:
-        """
-        if self.server is None:
-            logger.warning("Server is not running")
-            return
-
-        self.server.kill()
-        self.server = None
-
     def log(self, number: int = None, commit: str = None) -> OrderedDict:
```

Comment:
this failed for me in the demos unless number was type: str

Reply:
Oh, that's an easy `int(...)` wherever it gets passed into the args array which is in turn passed to `Popen`